.circleci/config.yml

```diff
@@ -331,6 +331,31 @@ jobs:
             docker push "tendermint/tendermint"
             docker logout
 
+  reproducible_builds:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run:
+          name: Build tendermint
+          no_output_timeout: 20m
+          command: |
+            sudo apt-get install -y ruby
```

Comment:
can't find where ruby is being used

Reply:
`gitian` is written in ruby:
https://github.com/devrandom/gitian-builder